sherlock/resources/data.json

```diff
@@ -2499,5 +2499,29 @@
     "urlMain": "https://www.zoomit.ir",
     "username_claimed": "kossher",
     "username_unclaimed": "noonewouldeverusethis7"
+  },
+  "koo": {
+    "errorMsg": "Invalid Profile",
+    "errorType": "message",
+    "url": "https://www.kooapp.com/profile/{}",
+    "urlMain": "https://www.kooapp.com",
+    "username_claimed": "john",
+    "username_unclaimed": "noonewouldeverusethis7"
+  },
+  "revish": {
```

Comment:
revish is down and gives us false positive. please remove revish from this PR and it could be merged.

Reply:
Resolved. Look into it. @TheYahya